projects/pgai/db/sql/incremental/029-add-vectorizer-name-column.sql

```diff
@@ -0,0 +1,45 @@
+alter table ai.vectorizer add column name text;
```

Comment:
```suggestion
alter table ai.vectorizer add column name name check (catalog_name ~ '^[a-z][a-z_0-9]*$');
```

Consider restricting the name to something that would make a reasonable postgres identifier that doesn't require double-quoting. There's a decent chance we may want to create other things in the future that include this name in the object name. It will also make the name "fit in" like other postgres objects.

I'm also torn on whether "name" is a good column name since "name" is a built-in type. I'd like to use it, since it's easy and succinct. We've done it before, so maybe it's ok.

Reply:
added